Dockerfile

```diff
@@ -40,7 +40,8 @@ LABEL \
     io.openshift.tags="cost,cost-management,prometheus,servicetelemetry,operators" \
     maintainer="Cost Management <cost-mgmt@redhat.com>" \
     name="koku-metrics-operator" \
-    summary="Koku Metrics Operator"
+    summary="Koku Metrics Operator" \
+    version="3.3.1"
```

Comment:
Is this version correct? Should we update a make command to ensure this is updated appropriately?

Reply:
Yes will add handling this via make cmd ensure correctness when updating what's new in the release.